db/migrate/20240830142019_add_differences_log.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_table :differences_logs do
+      primary_key :id
+      Date :date, null: false
```

Comment:
Should we make this less ambigious and just add an audit timestamp of created_at/updated_at as datetimes?

Reply:
The reason here is that I'm looking to detect logs for the same day which is much easier with just a date field rather than getting into messing about with ranges.

Reply:
Okay